src/js/components/LoginForm.js

```diff
@@ -81,21 +83,21 @@ var LoginForm = React.createClass({
         </div>
       );
     }
-
+      
     return (
       <Form className={classes.join(' ')} onSubmit={this._onSubmit}>
         {logo}
         {title}
         <fieldset>
           <FormField htmlFor="username" label={this.getGrommetIntlMessage('Username')}>
-            <input id="username" ref="username" type="email" />
+            <input id="username" ref="username" type={this.props.usernameType} />
           </FormField>
           <FormField htmlFor="password" label={this.getGrommetIntlMessage('Password')}>
             <input id="password" ref="password" type="password" />
           </FormField>
         </fieldset>
         {errors}
-        <Button className={CLASS_ROOT + "__submit"} primary={true} strong={true}
+        <Button id="login_button" className={CLASS_ROOT + "__submit"} primary={true} strong={true}
```

Comment:
I think it would be slightly more consistent to set the id to the same value as the class: "login-form__submit", rather than having a different naming convention.